cmp/grid/Grid.js

```diff
@@ -216,7 +216,11 @@ class LocalModel {
             rowDeselection: true,
             getRowHeight: () => this.rowHeight,
             getRowClass: ({data}) => model.rowClassFn ? model.rowClassFn(data) : null,
-            overlayNoRowsTemplate: model.emptyText || '<span></span>',
+            noRowsOverlayComponentFramework: hoistCmp({
+                render() {
+                    return model.emptyText;
+                }
+            }),
```

Comment:
Here I would suggest collapsing this to one line with the more minimal syntax `hoistCmp(() => model.emptyText)`